swagger.yaml

```diff
@@ -3152,638 +3152,4315 @@ paths:
-      image_type:
-        description: Type of image that should be generated.
-        $ref: '#/definitions/image_type'
-
-  assisted-service-iso-create-params:
-    type: object
-    properties:
-      openshift_version:
-        type: string
-        description: Version of the OpenShift cluster.
-      ssh_public_key:
-        type: string
-        description: SSH public key for debugging the installation.
-      pull_secret:
-        type: string
-        description: The pull secret obtained from Red Hat OpenShift Cluster Manager at cloud.redhat.com/openshift/install/pull-secret.
+          type: string
 
-  host-create-params:
+  connectivity-check-agent:
     type: object
-    required:
-      - host_id
     properties:
-      host_id:
+      agent_id:
         type: string
         format: uuid
-      discovery_agent_version:
-        type: string
+      nics:
+        type: array
+        items:
+          $ref: '#/definitions/connectivity-check-nic'
 
-  host_registration_response:
-    allOf:
-        - $ref: '#/definitions/host'
-        - properties:
-            next_step_runner_command:
-              type: object
-              description: Command for starting the next step runner
-              properties:
-                command:
-                  type: string
-                args:
-                  type: array
-                  items:
-                    type: string
-                retry_seconds:
-                  type: integer
-                  description: How long in seconds to wait before retrying registration if the command fails
+  connectivity-check-params:
+    type: array
+    items:
+      $ref: '#/definitions/connectivity-check-agent'
 
+  agent-list:
+    type: array
+    items:
+      $ref: '#/definitions/agent'
 
-  host:
+  host-list:
+    type: array
+    items:
+      $ref: '#/definitions/host'
+
+  v1cluster-create-params:
     type: object
     required:
-      - kind
-      - id
-      - href
-      - status
-      - status_info
+      - name
+      - openshift_version
+      - pull_secret
     properties:
-      kind:
-        type: string
-        enum: ['Host', 'AddToExistingClusterHost']
-        description: |
-          Indicates the type of this object. Will be 'Host' if this is a complete object or 'HostLink' if it is just a link, or
-          'AddToExistingClusterHost' for host being added to existing OCP cluster, or
-      id:
-        type: string
-        format: uuid
-        description: Unique identifier of the object.
-        x-go-custom-tag: gorm:"primary_key"
-      href:
-        type: string
-        description: Self link.
-      cluster_id:
+      name:
         type: string
-        format: uuid
-        x-go-custom-tag: gorm:"primary_key;foreignkey:Cluster"
-        description: The cluster that this host is associated with.
-      status:
+        minLength: 1
+        maxLength: 54
+        description: Name of the OpenShift cluster.
+      high_availability_mode:
```

Comment:
maybe we should add a separate definition to high_availability_mode, so that we won't have 2 enums (V2clusterHighAvaila.. and V1clusterHighAvail...) which actually have the same values.

Reply:
Does it matter now?  It will be clusterHighAvailabilityMode